src/lib/zork/index.ts

```diff
@@ -272,7 +270,6 @@ const onDataChannel = (ctx: Context, channel: any) => {
   // it forwards data for the SOCKS session.
   channel.onopen = () => ctx.log(`[channel ${channelId}] datachannel opened`);
   initSocksSessionAndFwdSocket(ctx, channel);
-  channel.onmessage = (event: any) => ctx.socksSession.handleDataFromSocksClient(event.data);
```

Comment:
Not storing any socks sessions on the context doesn't work if we need to refer to one of them later, as we now do in channel.onmessage. Storing them in a map is the way to go (see https://github.com/uProxy/uproxy/pull/2859/files#r92925469).